pkg/rhel/rhel-version.go

```diff
@@ -0,0 +1,51 @@
+package rhel
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/openshift-knative/hack/pkg/prowgen"
+)
+
+func ForSOBranchName(soBranchName string) (string, error) {
+	soConfig, loadErr := prowgen.LoadConfig("config/serverless-operator.yaml")
```

Comment:
if we don't embed the file the path might not always be there

Reply:
Uh. Yes. Updated